masonry_core/src/passes/update.rs

```diff
@@ -383,17 +383,20 @@ fn update_stashed_for_widget(
             .update(&mut ctx, &mut props, &Update::StashedChanged(stashed));
         state.item.is_stashed = stashed;
         state.item.needs_update_focus_chain = true;
-        // Note: We don't need request_repaint because stashing doesn't actually change
-        // how widgets are painted, only how the Scenes they create are composed.
-        state.item.needs_paint = true;
+
+        // Stashing doesn't stop the accessibility node from being generated or updated
+        // but it changes an accessibility flag.
```

Comment:
I thought it was supposed to stop it....

Am I better off just not including stashed children in the children_ids from a widget or something?

Reply:
> I thought it was supposed to stop it....

So far the behavior we have is that stashed widgets are included in the accessibility tree, so this is just better documenting existing code.

But I've started a discussion on Zulip about just not generating accessibility nodes for stashed widgets, which should be its own PR.